curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

```diff
@@ -14,6 +14,8 @@ Each new term in the Fibonacci sequence is generated by adding the previous two
 <div style='text-align: center;'>1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...</div>
 
 By considering the terms in the Fibonacci sequence whose values do not exceed <code>n</code>, find the sum of the even-valued terms.
+
+For example, if <code>n = 8</code>, your function should sum the even-valued Fibonacci numbers between <code>1 and 8</code>, inclusive. So, <code>fiboEvenSum(8)</code> should return <code>10</code>.
```

Comment:
```suggestion
For example, if <code>n = 8</code>, your function should sum the even-valued Fibonacci numbers between <code>1</code> and <code>8</code>, inclusive. So, <code>fiboEvenSum(8)</code> should return <code>10</code>.
```

Reply:
no problem. I'll change that. I only speak English though. Will you open another issue for the translation then

Reply:
Let me clarify: What happens with challenges is anything that is not inside `<code></code>` tags, can be automatically translated (hopefully), or contributors in the other languages, know to manually translate elements not inside those tags.

Reply:
ok. I see. I'll use Google translate to change the text outside the `<code></code>` tag then. I'll update the PR when I'm done.

Reply:
No, you do not need to do that. The `code` tags just assist the other contributors. You do not need to translate anything for this PR.

Reply:
Ok. Thanks for clarifying. I'll push the correction to your suggested changes now